### DISCUSSION TITLE
Data source: grocery flyers❓

### DISCUSSION THREAD

**Discussion by TechShooter** [openfoodfacts/open-prices#312] [Q&A]
Does grocery flyers count for prices? Can I upload them as proof?

**Comment by raphodn** [comment]
Hi @TechShooter , could you send an exemple ?

We had a similar question a few months ago at the beginning of the project, for now we don't consider it as a valid price proof (the 3 valid price proof being a price tag in the shop ; a receipt ; a gdpr request).

The tricky bits : 
- is the location specified on the flyer ?
- is the date specified ?
- most likely no barcode = risk of selecting the wrong OFF product

But we could think about allowing alternative price proofs to test out :+1: we would need to add a new proof type beforehand, to clearly label them.

**Reply by raphodn** [reply]
somewhat related issue : #244
